docs/CodeDevelopment/compile.md

```diff
@@ -63,13 +63,13 @@ the Pull Requests gates are implemented and enforced.
 
 ???+ info
     - In Project Mu repos the config file is generally at ```.pytool/CISettings.py```
-    - Project Mu runs on Windows 10 using the following tags: VS2017 and VS2019
+    - Project Mu runs on Windows 10/Windows 11 using the following tags: VS2019 and VS2022
     - Project Mu runs on Ubuntu 18.04 using the tags: GCC5
```

Comment:
Ubuntu 18.04 is old at this point, do we want to update this or more generically specify Ubuntu/Linux distros?

Reply:
You are correct, 18.04 is old, but it is still supported.

https://github.com/actions/runner-images/tree/main/images

Our CI is setup to run on ubuntu latest, which is 22.04.1 LTS, according to the above runner-images documents. 

I will update to say latest ubuntu